### ISSUE TITLE
stress-ng doesn't work on fresh install

### ISSUE THREAD

**Issue by mikeazo** [phoronix-test-suite/phoronix-test-suite#184]
I am on an Ubuntu 16.04 machine where I installed PTS with apt. I'm trying to run stress-ng and it is not working. Running it normally, I just get a message that "This test failed to run properly." Running it with *debug-run*, I get this:

```
========================================
Stress-NG (Run 1 of 3)
========================================


Test Run Command: cd /home/mike/.phoronix-test-suite/installed-tests/pts/stress-ng-1.2.0/ && ./stress-ng -t 30 --metrics-brief --switch 0 2>&1

stress-ng: info:  [15915] dispatching hogs: 1 switch
stress-ng: info:  [15915] successful run completed in 30.00s
stress-ng: info:  [15915] stressor       bogo ops real time  usr time  sys time   bogo ops/s   bogo ops/s
stress-ng: info:  [15915]                           (secs)    (secs)    (secs)   (real time) (usr+sys time)
stress-ng: info:  [15915] switch         13959481     30.00      3.99     25.21    465311.37    478064.42

Result Parsing Search Key: stress-ng:


Result Line: 


Test Result Value: 


The test run did not produce a result.


Log File At: /home/mike/.phoronix-test-suite/installed-tests/pts/stress-ng-1.2.0/stress-ng-1.2.0-1499791647-1.log
```

Runs 2 and 3 produce similar results. It looks like it is getting results, but is not able to parse them correctly?

I also tried installing with the tar.gz file on the website and got the same results.

**Comment by tippettj** [comment]
Pull request up at https://github.com/phoronix-test-suite/test-profiles/pull/6

**Comment by tippettj** [comment]
@mikeazo, this issue has been resolved in the updated 1.2.1 release of pts/stress-ng test-profile and should be live on OpenBenchmarking.org.
